src/main/java/net/sf/jabref/model/EntryTypes.java

```diff
@@ -95,8 +98,11 @@ public void removeType(String name) {
 
     }
 
-    public static final InternalEntryTypes BIBTEX = new InternalEntryTypes(BibtexEntryTypes.MISC, BibtexEntryTypes.ALL, IEEETranEntryTypes.ALL);
-    public static final InternalEntryTypes BIBLATEX = new InternalEntryTypes(BibLatexEntryTypes.MISC, BibLatexEntryTypes.ALL);
+
+    public static final InternalEntryTypes BIBTEX = new InternalEntryTypes(BibtexEntryTypes.MISC,
+            Arrays.asList(BibtexEntryTypes.ALL, IEEETranEntryTypes.ALL));
```

Comment:
@koppor Some possible Heap pollution with the usage of the vargars parameter 
http://stackoverflow.com/questions/12462079/potential-heap-pollution-via-varargs-parameter

I tried removing the ... first but then I noticed that you have a List<List<EntryType>>
Because, Types.ALL returns already a List. so you have a list containig a List of EntryType